app/main.py

```diff
@@ -1,2 +1,46 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self, initial_capacity: int = 8) -> None:
+        self.capacity = initial_capacity
+        self.size = 0
+        self.storage = [None] * self.capacity
+
+    class Node:
+        def __init__(self, key: Any, value: Any) -> None:
+            self.key = key
+            self.value = value
+            self.next = None
+
+    def __setitem__(self, key: Any, value: Any) -> None:
```

Comment:
```suggestion
    def __setitem__(self, key: Hashable, value: Any) -> None:
```
please fix everywhere